src/v2/view-builder/views/shared/ChallengeAuthenticatorDataPushOnlyFormView.js

```diff
@@ -1,15 +1,22 @@
-import { loc } from 'okta';
+// Common view for OV push and custom push.
```

Comment:
and same here. Let's see if we can have a prefix that these 3 files can share, so we know these are for OV and CustomApp authenticators. Adding something like `OTPPush` or `OVCustomApp` to the name will help... this specific file is for `PushOnly` that is why I was suggesting `OTPPush` for the others, but up to you on what a good name could be